.cljfmt.edn

```diff
@@ -213,6 +213,7 @@
   metabase.sync.util/with-start-and-finish-logging                                     [[:default]]
   metabase.sync.util/with-start-and-finish-logging*                                    [[:default]]
   metabase.sync.util/with-sync-events                                                  [[:default]]
+  metabase.task/rerun-on-error                                                         [[:block 1]]
```

Comment:
Very annoying that I had to add this manually. I don't recall having to do this before.

Feels like I'm doing something wrong. @camsaul